duckduckgo/macos-browser
# Update BSK for iOS Autofill on by default changes

Task/Issue URL: https://app.asana.com/0/0/1205296228305939/f
Tech Design URL:
CC:

**Description**:
BSK update for new iOS autofill subfeature flag

**Steps to test this PR**:
1. There should be no impact to app at all so just launch and confirm access to Autofill etc all looks normal



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
